src/app/student-login/student-login.component.html

```diff
@@ -1,16 +1,16 @@
+<form [ FormGroup ]="loginForm"(ng submit="login()")>
```

Comment:
(ngSubmit) = "login()"

Check the format

Reply:
changed